src/vars/default_certificates.yml

```diff
@@ -1,4 +1,5 @@
 ---
+certificates_ca_directory: /root/certificates
```

Comment:
this is needed as otherwise you can't reference the certs w/o including the certs role